src/classes/HiFiMixerSession.ts

```diff
@@ -198,6 +199,60 @@ export class HiFiMixerSession {
         });
     }
 
+    /**
+     * Sends the command `audionet.personal_volume_adjust` to the mixer. This adjusts the volume of another user session, only from the perspective of this client, for the current connection only.
+     * 
+     * @returns If this operation is successful, the Promise will resolve with `{ success: true, audionetAdjustPersonalVolumeResponse: <The response to `audionet.personal_volume_adjust` from the server in Object format> }`.
+     * If unsuccessful, the Promise will reject with `{ success: false, error: <an error message> }`.
+     */
+    async adjustPersonalVolume(visitIdHash: String, gain: Number): Promise<any> {
```

Comment:
Instead of returning `Promise<any>` here, please create a new `interface` corresponding to the structure of the data returned, then return `Promise<[that interface]>`.

For example, you may create a new interface at the top of the file like:
```ts
export interface AdjustPersonalVolumeResponse {
    success: boolean,
    error?: string,
    audionetAdjustPersonalVolumeResponse?: AudionetAdjustPersonalVolumeResponse
}
```

you'll then need another interface like:

```ts
interface AudionetAdjustPersonalVolumeResponse {
    <the data returned from the server>
}
```

If you don't know how `interface AudionetAdjustPersonalVolumeResponse` would be structured, first ask yourself if there's a good reason why that might be. if you're _sure_ there's a good reason, then you don't need that second interface, and you can replace `audionetAdjustPersonalVolumeResponse?: AudionetAdjustPersonalVolumeResponse` in the first `interface` with `audionetAdjustPersonalVolumeResponse?: any`.

in general, it's good practice to use `interface`s instead of `any` whenever possible. anytime there's an `any`, there's an increased chance of error.